CODEOWNERS

```
@@ -0,0 +1,4 @@
+# This file was automatically generated by the Apollo SecOps team
+# Please customize this file as needed prior to merging.
+
+* @apollographql/atlas @apollographql/polaris
```

Probably just need the apollo-rs team rather than the whole router team as well

```suggestion
* @apollographql/atlas @apollographql/rust-platform
```